javascript_nodejs/50.contoso-cafe-bot/index.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+ 
+const path = require('path');
+const restify = require('restify');
+
+const BOT_CONFIGURATION_ERROR = 1;
+
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter, MemoryStorage, ConversationState, UserState, BotStateSet } = require('botbuilder');
+
+// This bot's main dialog.
+const Bot = require('./bot');
```

Comment:
>Bot [](start = 6, length = 3)

While its technically fine to have a default export I wonder if we should have this be a named export so that you use the more consistent `const { Bot } = require('./bot');` to import it. I wasn't sure if it was a class or variable at first glance.